docs/reference/tctl/batch.md

```diff
@@ -7,33 +7,15 @@ tags:
   - tctl
 ---
 
-<!-- prettier-ignore -->
-import * as WhatIsAWorkflowExecution from '../../content/what-is-a-workflow-execution.md'
-
-The `tctl batch` commands operate batch jobs. A batch job can signal, cancel, or terminate <preview page={WhatIsAWorkflowExecution}>Workflow Executions</preview>.
+The `tctl batch` commands enable batch jobs. A batch job can signal, cancel, or terminate [Workflow Executions](/docs/content/what-is-a-workflow-execution).
 
 Terminating a batch job does not roll back the operation performed by the batch job. However, you can use `tctl workflow reset` to roll back Workflow Executions.
 
-## `start`
-
-import TctlBatchStart from './batch/start.md'
-
-<TctlBatchStart/>
-
-## `list`
-
-import TctlBatchList from './batch/list.md'
-
-<TctlBatchList/>
-
-## `describe`
-
-import TctlBatchDescribe from './batch/describe.md'
-
-<TctlBatchDescribe/>
-
-## `terminate`
+- [`tctl batch start`](./batch/start)
+- [`tctl batch list`](./batch/list)
+- [`tctl batch describe`](./batch/describe)
+- [`tctl batch terminate`](./batch/terminate)
```

Comment:
We want all of these to be in `/docs/reference/tctl/batch/start` format so we can automate replacement later on.